spec/requests/user_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
```

Comment:
- [ ] We should use `type: request` as well done in the `spec/request/post_controler_spec.rb`. You can check this [article](https://www.microverse.org/blog/understanding-rspec-controller-request-specs-integration-test-using-capybara#toc-controller-spec-http-request) for help.

![image](https://github.com/Otegaa/blog-app/assets/58642949/73e41fb3-5a17-462f-bd82-d8090a6294ed)